src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java

```diff
@@ -240,10 +246,18 @@ private void processScannerResult(final QueryManager qm, final Component compone
     private Set<Vulnerability> syncVulnerabilities(final QueryManager qm, final ScanKey scanKey, final ScannerResult scannerResult) {
         final var syncedVulns = new HashSet<Vulnerability>();
 
+        final Component component = withJdbiHandle(handle ->
```

Comment:
Any reason for re-fetching the component that was already fetched earlier? https://github.com/DependencyTrack/hyades-apiserver/blob/a8a454a6270bb9cb6163397b7d948d3b2f4499cf/src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java#L170